Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -368,6 +370,13 @@
         'Era' : 'Phase2C10_timing_layer_bar',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFullTrigger','RecoFullGlobal', 'HARVESTFullGlobal'],
     },
+    '2026D48' : {
+        'Geom' : 'Extended2026D48',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T15',
```

Comment:
@kpedro88 there is no need for it at the moment. As the list of `DetId`s between `T15` and `T16` is the same, this choice is technically sound.
I cannot exclude a later update in case one discovers there is a better choice set of parameters but this will be checked during the validation procedure.